pkgs/tools/package-management/packagekit/default.nix

```
@@ -39,8 +39,9 @@ stdenv.mkDerivation rec {
     "--disable-offline-update"
     "--localstatedir=/var"
     "--sysconfdir=/etc"
-    "--with-dbus-sys=${placeholder "out"}/share/dbus-1/system.d"
-    "--with-systemdsystemunitdir=${placeholder "out"}/lib/systemd/system/"
+    "--with-dbus-sys=${placeholder "out"}/etc/dbus-1/system.d"
```

```suggestion
    "--with-dbus-sys=${placeholder "out"}/share/dbus-1/system.d"
```

the other dir is deprecated